### PR TITLE
Add option for autofill

### LINKED ISSUE #146: Make autofill compliant with AMO policy
<Do not use this to ask about lost codes or accounts>
<More info here: https://github.com/Authenticator-Extension/Authenticator/wiki/Lost-Codes>

### PR DESCRIPTION
**Pull request by mymindstorm**
closes #146

### CODE REVIEW

File: src/ui/menu.ts [hunk 1]
```diff
@@ -103,6 +104,12 @@ async function menu(_ui: UI) {
           return false;
         }
       },
+      saveAutofill: () => {
+        localStorage.autofill = _ui.instance.useAutofill;
+        useAutofill =
+            (localStorage.autofill === 'true') ? true : false || false;
```
Comment (Sneezry) [inline src/ui/menu.ts:110]: `useAutofill = (localStorage.autofill === 'true')` is enough.

**Review by Sneezry — APPROVED** [review]

**Comment by gek15** [comment]
U

**Comment by gek15** [comment]
O

**Comment by gek15** [comment]
O

**Comment by mymindstorm** [comment]
So if you ban someone, you can only collapse their comments? And you have to unban them temporarily to delete comments again? What sense does any of that make?

**Comment by Sneezry** [comment]
It seems we can block someone to access to the org: https://help.github.com/articles/blocking-a-user-from-your-organization/

**Comment by mymindstorm** [comment]
Already done a while ago, it just won't let you delete comments from someone you banned for some reason. I had to go back and unban to delete all the spam. Makes no sense at all to me.

**Pull request merged**